docker/nginx.conf

```diff
@@ -26,27 +26,27 @@ http {
 
         location / {
             proxy_pass http://127.0.0.1:4445;
+            proxy_set_header Host      $host;
+            proxy_set_header X-Real-IP $remote_addr;
```

Comment:
If I remove lines 29 and 30, BrowserStack works again.
The dashboard redirect also works.

Is there a reason to include them?

Reply:
OK, I'll remove them and push again.